docs/reference/query-dsl/mlt-query.asciidoc

```diff
@@ -241,6 +241,11 @@ number of terms that must match.
 The syntax is the same as the <<query-dsl-minimum-should-match,minimum should match>>.
 (Defaults to `"30%"`).
 
+`fail_on_unsupported_term`::
+Controls whether the query should fail if it encounters a term that is not
+`text` or `keyword' (true) or ignore the error and continue (false). Defaults to
```

Comment:
> ignore the error and continue

sorry to nit here, but it feels misleading to say that it will ignore the error, since there will not be an error at all.

For example, if `fields = ["foo", "bar", "baz"]` and `"bar"` is a numeric, then `"bar"` will simply be ignored and it will be the same as if `fields = ["foo", "baz"]`.